### ISSUE TITLE
Decide to use design sprints and come up with a structure

### ISSUE THREAD

**Issue by menor** [resmio/git-along#24]
They are useful when building new features. @fabiennepimenta can probably explain it better than me. [Here](https://developers.google.com/design-sprint/downloads/DesignSprintMethods.pdf)'s a good overview I found though.

The issue is about coming up with a structure and a way to integrate them in our flow.

**Comment by fabiennepimenta** [comment]
I've made a lot of research today about this topic and my conclusion is:  

### Yes, we should do it, but only for big problems.

Specially because it takes one entire week ( with several people ) and this is mostly used to define how to answer to big / important questions. 

If you still don't know what a "Design Sprint" is I can explain it to you really quickly or you can check this: [here](http://www.gv.com/sprint/)


</br>

### "A sprint helps you solve big problems and test new ideas in just **five** days"

It's used to avoid to spend time building a feature that will not be used or has no value ( but we believed that our users would love). The process is that we build an idea and we have 5 days to come up with a prototype and test it. If it works, we can start developing. 

![image](https://cloud.githubusercontent.com/assets/3285116/18674854/db66b456-7f50-11e6-8e86-9a42e5a741de.png)

</br>

One thing that we can do is about the issue that we have ( and one of the most important ones) 

### "How to engage our users in less that 30 days? "

**Comment by menor** [comment]
Grande Trabalho. Obrigado

What do you think about adding a `design sprint` tag to our size labels?, we can discuss what needs to be turned into a sprint in the estimating meeting.

**Comment by fabiennepimenta** [comment]
Yes, we can do that. The problem is that "Design sprints" will not happen that often and actually I'm not sure if we have enough people to actually do it. Yesterday after this comment I also realise that usually this "team" for the sprint should always be composed by more than 1 designer.  

Yesterday night I ask for some inputs and I think the best solution for us is actually "design studios" that are only during a few hours ( 2 or 3 ) and it's to generate ideas – that's what we need for the engagement problem, and I have a few ideas how to do it :) 

First: I will make some assumptions about why we don't have engagement. 

Second: I will ask CR to make the some questions to our customers to validate this assumptions

Third: Design Studio to bring new ideas about how to do it (with some inputs from customers already).  


I think is the best solution for us, honestly. @menor what do you think? The good thing is that the "idea - to - learn" concept will be also used here. :)

**Comment by menor** [comment]
Great solution. I think that it's faster and more simpler and that it's always good. ;)
Is OK if l add your comment to the readme.md and create a `design_studio` label?
Thank you!

**Comment by fabiennepimenta** [comment]
yap, sure :)

**Comment by ihucos** [comment]
My opinion is, if @fabiennepimenta can guide us through the process we can try it and see if it makes sense for us.
